app/components/graph-editor/drawable-interfaces.ts

```diff
@@ -61,6 +55,24 @@ export interface DrawableGraph {
     readonly edges: Iterable<DrawableEdge>;
 
     /**
+     * selectedItems  
+     *   The set of selected graph elements.
+     */
+    readonly selectedItems: Set<Drawable>;
```

Comment:
I did not know that you could make interfaces have readonly stuff. We should probably use this features of ts more often throughout the project.